salt/modules/pcs.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+'''
+Pcs Command Wrapper
+========================
+
+The pcs command is wrapped for specific functions
+
+:depends: pcs
+'''
+from __future__ import absolute_import
+
+# Import python libs
+
+# Import salt libs
+import salt.utils
+
+
+def __virtual__():
+    '''
+    Only load if pcs is installed
+    '''
+    if salt.utils.which('pcs'):
+        return 'pcs'
+    return False
+
+
+def auth(nodes, pcsuser='hacluster', pcspasswd='hacluster', **kwargs):
```

Comment:
If we're going to document the use of `extra_args` let's use that explicitely versus using `kwargs`